src/Models.elm

```diff
@@ -0,0 +1,9 @@
+module Models exposing (..)
```

Comment:
Is there a name that would map better to the domain of the application vs the content of the file?

Reply:
I know! `Models` is a temporary name, I'm looking into changing into something more accurate.